docs/installation/helm.md

```diff
@@ -119,25 +237,44 @@ To run tests, try:
   $ helm test siglens
 ...
 ```
-#### Step 6: Accessing SigLens
+
+### Step 6: Accessing SigLens
 SigLens is now installed in your local Minikube cluster. To access the SigLens UI, follow these steps:
-Forward local port 8081 to the ingest service using: 
- ```
-$ kubectl port-forward svc/siglens-ingest-svc 8081:8081
-```
-Get data in SigLens by loading sample data, using our supported ingestion methods, or integrating with your existing tools.
 
-Forward local port 8000 to the UI / query service using:
+1. Forward local port 8081 to the ingest service using:
+
+```bash
+kubectl port-forward svc/siglens-ingest-svc 8081:8081
 ```
- $ kubectl port-forward svc/siglens-query-svc 8000:8000
+
+2. Get data in SigLens by loading sample data, using our supported ingestion methods, or integrating with your existing tools. Forward local port 8000 to the UI / query service using:
+   
+```bash
+kubectl port-forward svc/siglens-query-svc 8000:8000
```

Comment:
we shuold keep the port same here (5122) for UI as it is in the rest of the documentation